benchmarks/dynamo/common.py

```diff
@@ -247,30 +247,40 @@ def tensor_is_on_xla(tensors):
     return any(map(lambda x: x.device.type == "xla", tensors))
 
 
-def timed(model, model_iter_fn, example_inputs, times=1, return_result=False):
+def timed(model, model_iter_fn, example_inputs, times=1, return_result=False, collect_outputs=False):
+    use_xla = tensor_is_on_xla(example_inputs)
     synchronize()
-    if tensor_is_on_xla(example_inputs):
+
+    reset_rng_state()
+    if use_xla:
         import torch_xla.core.xla_model as xm
```

Comment:
could you do the import once at the top of the function and not have to repeat it?

Reply:
well i was also thinking of just doing the import once inside the 'timed' function at the top when you check 'use_xla' var the first time